packages/workers-shared/asset-server-worker/src/global.d.ts

```diff
@@ -0,0 +1,13 @@
+type Environment = "production" | "local";
+
+type Env = {
+	// ASSETS_MANIFEST is a pipeline binding to an ArrayBuffer containing the
+	// binary-encoded site manifest
+	ASSETS_MANIFEST: ArrayBuffer;
+
+	// ASSETS_KV_NAMESPACE is a pipeline binding to the KV namespace that the
+	// assets are in.
+	ASSETS_KV_NAMESPACE: KVNamespace;
+
+	ENVIRONMENT: Environment;
```

Comment:
Not needed?

Reply:
indeed. I meant to remove it and got distracted. will remove